[2022-08-06 19:38:11,713] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 19:38:11,723] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 19:38:11,723] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:11,724] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:11,724] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:11,737] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-28 00:00:00+00:00
[2022-08-06 19:38:11,742] {standard_task_runner.py:52} INFO - Started process 2110 to run task
[2022-08-06 19:38:11,744] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-28T00:00:00+00:00', '--job-id', '644', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmptvb_3atj', '--error-file', '/tmp/tmpkpu12glp']
[2022-08-06 19:38:11,744] {standard_task_runner.py:80} INFO - Job 644: Subtask second_task
[2022-08-06 19:38:11,806] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:11,879] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-28T00:00:00+00:00
[2022-08-06 19:38:11,880] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:11,881] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:11,891] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:11,892] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:11,892] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:11,919] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220228T000000, start_date=20220806T193811, end_date=20220806T193811
[2022-08-06 19:38:11,955] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:11,989] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:05,413] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,428] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 21:52:05,429] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,429] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:05,429] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:05,453] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-28 00:00:00+00:00
[2022-08-06 21:52:05,461] {standard_task_runner.py:52} INFO - Started process 2309 to run task
[2022-08-06 21:52:05,467] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-28T00:00:00+00:00', '--job-id', '646', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp91egwduk', '--error-file', '/tmp/tmpurncux9b']
[2022-08-06 21:52:05,467] {standard_task_runner.py:80} INFO - Job 646: Subtask second_task
[2022-08-06 21:52:05,558] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:05,655] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-28T00:00:00+00:00
[2022-08-06 21:52:05,656] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:05,657] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:05,673] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:05,675] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:05,675] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:05,711] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220228T000000, start_date=20220806T215205, end_date=20220806T215205
[2022-08-06 21:52:05,757] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:05,813] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:29,230] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,246] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 21:57:29,246] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,246] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:29,246] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:29,262] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-28 00:00:00+00:00
[2022-08-06 21:57:29,269] {standard_task_runner.py:52} INFO - Started process 2103 to run task
[2022-08-06 21:57:29,272] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-28T00:00:00+00:00', '--job-id', '645', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpcdsoi_b6', '--error-file', '/tmp/tmpfk548zkd']
[2022-08-06 21:57:29,273] {standard_task_runner.py:80} INFO - Job 645: Subtask second_task
[2022-08-06 21:57:29,369] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:29,506] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-28T00:00:00+00:00
[2022-08-06 21:57:29,507] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:29,508] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:29,529] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:29,531] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:29,532] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:29,640] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220228T000000, start_date=20220806T215729, end_date=20220806T215729
[2022-08-06 21:57:29,685] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:29,723] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:32,186] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,206] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 22:12:32,206] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,206] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:32,206] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:32,230] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-28 00:00:00+00:00
[2022-08-06 22:12:32,239] {standard_task_runner.py:52} INFO - Started process 2118 to run task
[2022-08-06 22:12:32,243] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-28T00:00:00+00:00', '--job-id', '647', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpd_gl4t99', '--error-file', '/tmp/tmpkievlbun']
[2022-08-06 22:12:32,243] {standard_task_runner.py:80} INFO - Job 647: Subtask second_task
[2022-08-06 22:12:32,351] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:32,462] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-28T00:00:00+00:00
[2022-08-06 22:12:32,463] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:32,463] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:32,481] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:32,483] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:32,483] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:32,516] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220228T000000, start_date=20220806T221232, end_date=20220806T221232
[2022-08-06 22:12:32,575] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:32,633] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:53,021] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,040] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [queued]>
[2022-08-06 22:23:53,040] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,040] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:53,040] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:53,065] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-02-28 00:00:00+00:00
[2022-08-06 22:23:53,072] {standard_task_runner.py:52} INFO - Started process 2170 to run task
[2022-08-06 22:23:53,076] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-02-28T00:00:00+00:00', '--job-id', '647', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmptyqiv8cz', '--error-file', '/tmp/tmpph8gu2vc']
[2022-08-06 22:23:53,076] {standard_task_runner.py:80} INFO - Job 647: Subtask second_task
[2022-08-06 22:23:53,155] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-02-28T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:53,254] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-28T00:00:00+00:00
[2022-08-06 22:23:53,255] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:53,255] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:53,272] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:53,273] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:53,274] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:53,315] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220228T000000, start_date=20220806T222353, end_date=20220806T222353
[2022-08-06 22:23:53,368] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:53,405] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
